src/components/common/LikelyToGiveSelect/LikelyToGiveSelect.test.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { LikelyToGiveEnum } from 'src/graphql/types.generated';
+import { LikelyToGiveSelect } from './LikelyToGiveSelect';
+
+jest.mock('src/utils/functions/getLocalizedLikelyToGive', () => ({
+  getLocalizedLikelyToGive: (_t: any, val: string) => `${val}`,
+}));
+
+describe('LikelyToGiveSelect', () => {
+  it('should render a Select with all LikelyToGiveEnum options', async () => {
+    render(<LikelyToGiveSelect value="" onChange={() => {}} />);
+
+    const select = screen.getByRole('combobox');
+    await userEvent.click(select);
```

Comment:
In most of our tests, we pull the `getBy...` functions from the `render` return value.

Also, I usually don't assign elements that it found to a variable (like `select`) unless it's used in multiple places to keep the test a little shorter.

```suggestion
    const { getByRole } = render(<LikelyToGiveSelect value="" onChange={() => {}} />);

    await userEvent.click(getByRole('combobox'));
```